client/deposits/list/index.js

```diff
@@ -28,6 +26,7 @@ import DetailsLink, { getDetailsURL } from 'components/details-link';
 import ClickableCell from 'components/clickable-cell';
 import Page from '../../components/page';
 import DepositsFilters from '../filters';
+import DownloadButton from '../../components/download-button';
```

Comment:
This can be:
```suggestion
import DownloadButton from 'components/download-button';
```

Reply:
This is cool! It indeed works. But how? 😄

Reply:
I've committed the change.